File: FamilyTools.Web/src/app/models/account-line.ts
import { BaseModel } from './base-model';
import { User } from './user';

export class AccountLine extends BaseModel {
  userLink: User;
  value: number;

  constructor(userLink: User, value: number, id?: number, creationDate?: Date, updateDate?: Date){
    super(id, creationDate, updateDate);
    this.userLink = userLink;
    this.value = value;
  }
}

File: FamilyTools.EasyCompta/Model/AccountLine.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace FamilyTools.EasyCompta.Model
{
    [Table("AccountLines")]
    public class AccountLine: BaseModel
    {

        public string Name { get; set; }

        public User UserLink { get; set; }

        public float Value { get; set; }

    }
}
